README.md

```diff
@@ -1,5 +1,15 @@
 # GOV.UK coronavirus - find support
 
+⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
+
+<p style="text-align: center">This repository is the old Coronavirus Find Support service - https://find-coronavirus-support.service.gov.uk.</p>
+
+<p style="text-align: center; font-weight: bold;">It is no longer deployed or in active development.</p>
+
+<p style="text-align: center">The service was migrated to [Smart Answers](https://github.com/alphagov/smart-answers) and is now available here: https://www.gov.uk/find-coronavirus-support</p>
```

Comment:
Nit: Maybe worth putting a link the flow definition in smart answers rather than just the root?

Reply:
@theseanything - thanks - fixed